lib/nonce.js

```diff
@@ -0,0 +1,37 @@
+const CSP_HEADER = 'Content-Security-Policy';
+const CSPRO_HEADER = 'Content-Security-Policy-Report-Only';
```

Comment:
I see these same strings are used in `pxcdenforcer.js` -- I recommend defining them in `utils/constants.js` and then importing them in the places they're used